microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
 
-First, you must set up the site so that you can make changes to it later.
+1. Open the **Lighthouse** tool.  If you don't see the tool in the main toolbar, click the **More tools** (![More tools](../media/more-tools-icon-light-theme.png)) button.
 
-1. [Open the source code for the site](https://glitch.com/edit/#!/tony).  This tab is referred to as the **editor tab**.
+1. Only keep the **Performance** category selected and clear all other categories. And keep all other options set by default. Here's an explanation of the different options:
 
-   ![The editor tab](../media/speed-glitch-tony-server-js.msft.png)
+   *  **Mode**: set this option to **Navigation (default)** to run tests when the webpage loads, or set the option to **Timespan** to run tests during a given period of time, or set the option to **Snapshot** to run tests on the webpage as it appears now.
 
-1. Click **tony**.  A menu appears.
+   *  **Device**: set this option to **Mobile** to simulate a mobile user agent string and a mobile viewport, or set the option to **Desktop** to test the webpage without any simulation.
 
-   ![The menu that appears after clicking 'tony'](../media/speed-glitch-tony-server-js-remix-project.msft.png)
+   *  **Categories**: this option allows you to only run a subset of the tests available in **Lighthouse**.
 
-1. Click **Remix Project**.  The name of the project changes from **tony** to a randomly generated name.  You now have your own editable copy of the code.  Later on, you can make changes to this code.
+1. Click **Analyze page load**:
 
-1. Click **Show** and then select **In a New Window**.  The demo opens in a new tab.  This tab is be referred to as the **demo tab**.  It may take a while for the site to load.
+    ![The Lighthouse tool](./images/lighthouse-tool.png)
 
-   ![The demo tab](../media/speed-glitch-tony-show-live.msft.png)
+1. After 10 to 30 seconds, a report of the performance of the site appears:
 
-1. Press `Ctrl`+`Shift`+`J` (Windows, Linux) or `Command`+`Option`+`J` (macOS).  DevTools opens alongside the rendered demo webpage.
+   ![The report for the Lighthouse tool about the performance of the site](./images/lighthouse-report.png)
 
-   ![DevTools and the demo](../media/speed-glitch-tony-show-live-console.msft.png)
+#### Handling report errors
 
-For the rest of the screenshots in this tutorial, DevTools is shown in a separate window.  Press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS) to open the Command Menu, type `Undock`, and then select **Undock into separate window**.
+If you ever get an error in your **Lighthouse** report, try running the tool again from an **InPrivate** window with no other tabs open. Running **Lighthouse** from an **InPrivate** window ensures that you're running Microsoft Edge from a clean state.  Microsoft Edge Extensions in particular often interfere with the auditing process.
```

Comment:
```suggestion
If you get errors in your **Lighthouse** report, try running the tool again from an **InPrivate** window with no other tabs open. Running **Lighthouse** from an **InPrivate** window ensures that you're running Microsoft Edge from a clean state.  Browser extensions can interfere with the auditing process.
```

Reply:
maybe reword from colloquialism metaphor "clean slate" to more direct technical, eg "with a clean baseline environment"